optimum/habana/transformers/generation/configuration_utils.py

```diff
@@ -31,5 +31,6 @@ def __init__(self, **kwargs):
         self.static_shapes = kwargs.get("static_shapes", None)
         self.ignore_eos = kwargs.get("ignore_eos", None)
         self.attn_softmax_bf16 = kwargs.get("attn_softmax_bf16", None)
+        self.bucketsize = kwargs.get("bucketsize", -1)
```

Comment:
```suggestion
        self.bucket_size = kwargs.get("bucket_size", -1)
```
Let's call it `bucket_size` to be consistent with `batch_size` and others.